native/cocos/renderer/pipeline/PlanarShadowQueue.cpp

```diff
@@ -42,7 +40,8 @@ namespace cc {
 namespace pipeline {
 
 PlanarShadowQueue::PlanarShadowQueue(RenderPipeline *pipeline)
-: _pipeline(pipeline) {
+: _phaseID(getPhaseID("planar-shadow")) {
+    _pipeline = pipeline;
```

Comment:
Why initialize _pipeline here?

Reply:
The weak pointer reference _pipeline is needed to get the pipelineSceneData from _pipeline below.

Reply:
Why can not initialize it in parameter list?

Reply:
> 

Referring to the ShadowMapBatchedQueue.cpp file, I wanted to be consistent in the style of my code.